conda/base/context.py

```diff
@@ -1185,7 +1185,7 @@ def reporters(self) -> tuple[Mapping[str, str]]:
             return (
                 {
                     "backend": "json" if self.json else "console",
-                    "output": "stdout",
+                    "stream": "stdout",
```

Comment:
Nah, that's configuring an output, which has a stream feature, we shouldn't confuse these terms